frontend/nc-games/src/components/navigation.jsx

```diff
@@ -11,20 +11,21 @@ export default function Navigation () {
         event.preventDefault();
         navigate("/reviewlist");
     };
-    // const toCategories = (event) => {
-    //     event.preventDefault();
-    //     navigate("/Categories/");
-    // };
+    const toCategories = (event) => {
+        event.preventDefault();
+        navigate("/Categories");
+    };
 
 
-    return (<div>
+    return (
+    <div>
         <h1>Welcome to NC games {name}</h1>
         <Link to="/reviewlist">
         <button onClick={toReviews}>Reviews</button>
         </Link>
-        {/* <Link to="/Categories">
+        <Link to="/Categories">
         <button onClick={toCategories}>Categories</button>
```

Comment:
This would be better named as something more descriptive of what it's doing e.g. 'handleClick'